views/lib/forms/paginatedForm.hbs

```diff
@@ -1,22 +1,26 @@
 <form class="form sections-{{sections}} {{class}}" action="{{action}}{{#if sso}}/sso/{{account}}{{/if}}" method="{{method}}" autocomplete="off" >
-  {{#for 1 (add sections 1) 1}}
-    <input id="section-{{this}}" type="radio" name="stage" {{#ifvalue this value=1}}checked="checked"{{/ifvalue}}/>
-  {{/for}}
+	{{#for 1 (add sections 1) 1}}
+		<input id="section-{{this}}" type="radio" name="stage" {{#ifvalue this value=1}}checked="checked"{{/ifvalue}}/>
+	{{/for}}
 
-  <div class="stages">
-    {{#for 1 (add sections 1) 1}}
-      <label for="section-{{this}}">{{this}}</label>
-    {{/for}}
-  </div>
+	<div class="stages">
+		{{#for 1 (add sections 1) 1}}
+			<label for="section-{{this}}">{{this}}</label>
+		{{/for}}
+	</div>
 
-  <span class="progressbar"></span>
+	<span class="progressbar"></span>
 
-  <div class="panels mb-2">
-    {{#block "panels"}}{{/block}}
-  </div>
+	<div class="panels mb-2">
+		{{#block "panels"}}{{/block}}
+	</div>
 
-  {{#block "buttons"}}{{/block}}
-  <button id="prevSection" class="btn btn-secondary" type="button" disabled="disabled">Zurück</button>
-  <button id="nextSection" class="btn btn-primary pull-right" type="submit" value="Submit"
-    data-next-label="Weiter" data-submit-label="{{#unless submit-label}}Absenden{{else}}{{submit-label}}{{/unless}}">Weiter</button>
+	{{#block "buttons"}}{{/block}}
+	<button id="prevSection" class="btn btn-secondary" type="button" disabled="disabled">Zurück</button>
+	{{#if isIE}}
+		<p style="margin-top:6px;"><i style="color: rgb(177, 4, 56);" class="fa fa-exclamation-triangle"></i> Die Registrierung kann nicht im Internet Explorer (unabhängig von der Version) durchgeführt werden. Bitte verwende einen anderen Browser, alle anderen Browser werden unterstützt. Firefox/Chrome können in den meisten Netzwerken auch ohne Administratorrechte installiert und benutzt werden.</p>
```

Comment:
```suggestion
		<p style="margin-top:6px;"><i style="color: rgb(177, 4, 56);" class="fa fa-exclamation-triangle"></i> Die Registrierung kann nicht im Internet Explorer (unabhängig von der Version) durchgeführt werden. Bitte verwende einen modernen Browser wie Firefox oder Chrome. Diese können in den meisten Netzwerken auch ohne Administratorrechte installiert und benutzt werden.</p>
```

Ich bin mir ziemlich sicher, dass wir nicht _"alle anderen Browser"_ unterstützen.*

**Fetch:**
[![image](https://user-images.githubusercontent.com/22987140/56374645-fbc61f00-6203-11e9-9cb6-06c80153baa1.png)](https://caniuse.com/#search=fetch)

(Im übrigen vermute ich, dass der Registrierungsprozess aufgrund von mangelndem `fetch` support im IE fehlschlägt.)

Reply:
Jap, könnte man vermutlich angehen aber Entwicklungszeit vs. Nutzen bzw. Nutzeranteil ist nicht optimal und auf Dauer trauert wohl keiner um IE. Edge (+Chromium) muss in Zukunft reichen.